bcgov/House-Innovations-HOUSNAV
# HOUSNAV-191: Results Page Updates

[HOUSNAV-191](https://hous-bssb.atlassian.net/browse/HOUSNAV-191)

## Overview & Purpose
Update the results page (and PDF output) to match the design and be able to handle content for more than 1 walkthrough

## Summary of Changes
- Updated results page to match design and handle more than 1 walkthrough
- Added related building types
- Updated PDF content component to handle more than 1 walkthrough and refactored for maintainability and readability

## Testing
- Complete walkthroughs of with varying building types and answers
- Make sure results page shows correct answers and amounts of answers
- Make sure PDF includes all appropriate questions, answers, and building code content

## Checklist
